client/js/ifml2code/client/elementrules/templates/xor-v.html.ejs

```diff
@@ -12,5 +12,8 @@
 <!-- /ko -->
 <% } -%>
 <% for (var i = 0; i < events.length; i += 1) { -%>
+  <% if (events[i].stereotype === 'system') { -%>
+    <c-<%= events[i].id %> params="fire: trigger.bind($data,'<%= events[i].id %>')"><c-<%= events[i].id %>>
```

Comment:
What do you think would it be better as name for this parameter `fire` or `trigger`?